manager/apis/app/v1alpha1/plotter_types.go

```diff
@@ -129,7 +129,7 @@ type Flow struct {
 
 	// Type of the flow (e.g. read)
 	// +required
-	FlowType DataFlow `json:"flowType"`
+	FlowType taxonomy.DataFlow `json:"flowType"`
```

Comment:
Removed the array of flows.  Now there is a single flow per datasetid.  FlowRequirements.StorageEstimate  won't get propagated to the plotter.  Rather the manager will use it to decide which modules to choose.  I.e. It will be passed as a parameter to the config policies.   FlowRequirements.IsNewDataSet is used to decide whether DataSetID is an existing entry in the catalog, or whether it indicates only the dest catalog / object name ... and thus Fybrik needs to create a new entry in the catalog.

Reply:
@roytman Plotter won't be aware of any of these. As I understand, storage estimate and new/existing dataset are used to determine the existing plotter arguments: where to allocate storage, data destination, etc.